changelog/unreleased/pull-3067

```diff
@@ -0,0 +1,10 @@
+Enhancement: Add options to configure Windows Shadow Copy Service
+
+Restic always used 120 sec. timeout and unconditionally created VSS snapshots
+for all volume mount points on disk. Now this behavior can be fine-tuned by
+new options, like exclude user specific volumes and mount points or completely
+disable auto snapshotting of volume mount points.
+
+For more details, see https://restic.readthedocs.io/en/stable/040_backup.html
```

Comment:
We should not link to specific pages in the manual (in case the manual and its URL structure changes later). If you are adding options in this PR, please sum these options up here.

Reply:
I get this idea from pull-2718, but you right, I should get rid of "For more details, see" clause.

Reply:
I updated your PR description to clarify what the vss.* things are (not everyone recognize `-o` options :) )

Reply:
Fixed in https://github.com/restic/restic/pull/3067/commits/1f23f4773e61580db82025d4fde118b5d5675691